trillian/ctfe/requestlog.go

```diff
@@ -34,37 +34,37 @@ const vLevel = 9
 type RequestLog interface {
 	// Start will be called once at the beginning of handling each request.
 	// The supplied context will be the one used for request processing and
-	// can be used by the logger to set values.
-	Start(context.Context)
+	// can be used by the logger to set values on the returned context.
```

Comment:
Does that mean that the returned context should be used in all the calls below?  If so, best to say so.

Reply:
It's set up by the handler code but I've added an explicit comment.